locales/nl-NL/src/create.ts

```diff
@@ -2,65 +2,73 @@ import {removeUndefinedLocaleResources} from 'sanity'
-  'start-in-create-action.label': undefined, // 'Start in Sanity Create'
+  'start-in-create-action.label': 'Begin in Sanity Create',
 
   /** CTA in "Start writing in Create" dialog: Continue to create */
-  'start-in-create-dialog.cta.continue': undefined, // 'Continue'
+  'start-in-create-dialog.cta.continue': 'Doorgaan',
   /** CTA in "Start writing in Create" dialog: Learn more */
-  'start-in-create-dialog.cta.learn-more': undefined, // 'Learn more'
+  'start-in-create-dialog.cta.learn-more': 'Meer leren',
   /** The details on what "Start writing in Create" implies */
-  'start-in-create-dialog.details': undefined, // 'This opens in a new window, linked directly to this document.'
+  'start-in-create-dialog.details':
+    'Dit opent in een nieuw venster, direct gekoppeld aan dit document.',
   /** Text for checkbox in "Start writing in Create" dialog for disabling the dialog in the future */
-  'start-in-create-dialog.dont-remind-me-checkbox': undefined, // 'Don’t remind me again'
+  'start-in-create-dialog.dont-remind-me-checkbox': 'Herinner mij hier niet meer aan',
   /** Toast error message when we dont have a resolved Sanity Create url*/
-  'start-in-create-dialog.error-toast.unresolved-url': undefined, // 'Unable to determine Sanity Create URL.'
+  'start-in-create-dialog.error-toast.unresolved-url': 'Kan de URL van Sanity Create niet bepalen.',
   /** Header of the "Start writing in Create" dialog */
-  'start-in-create-dialog.header': undefined, // 'Write naturally, publish structured'
+  'start-in-create-dialog.header': 'Schrijf op natuurlijke wijze, publiceer gestructureerd',
   /** The lede text in the "Start writing in Create" dialog */
-  'start-in-create-dialog.lede': undefined, // 'Create content your way in an AI-powered editor. Your writing automatically maps to the right fields in Sanity Studio as you type.'
+  'start-in-create-dialog.lede':
+    'Creëer inhoud op jouw manier in een door AI aangedreven editor. Je schrijfwerk wordt automatisch toegewezen aan de juiste velden in Sanity Studio terwijl je typt.',
 
   /** Text for the document pane banner informing users that the document is linked to Sanity Create */
-  'studio-create-link-banner.text': undefined, // 'This document is linked to Sanity Create'
+  'studio-create-link-banner.text': 'Dit document is gekoppeld aan Sanity Create',
 
   /** Unlink document from Sanity Create button text */
-  'unlink-from-create-button.text': undefined, // 'Unlink'
+  'unlink-from-create-button.text': 'Ontkoppelen',
 
   /** Unlink from Create dialog: Cancel button text */
-  'unlink-from-create-dialog.cancel.text': undefined, // 'Cancel'
+  'unlink-from-create-dialog.cancel.text': 'Annuleren',
   /** Unlink from Create dialog: Document title used if no other title can be determined */
-  'unlink-from-create-dialog.document.untitled.text': undefined, // 'Untitled'
+  'unlink-from-create-dialog.document.untitled.text': 'Zonder titel',
```

Comment:
```suggestion
  'unlink-from-create-dialog.document.untitled.text': 'Geen titel',
```

Not one hundred percent sure but i'm guessing it should be "No title" in stead of "Without title".

Reply:
When checking the other languages, it should indeed be "Geen titel".

Other languages: https://github.com/search?q=repo%3Asanity-io%2Flocales%20unlink-from-create-dialog.document.untitled.text&type=code